squad/core/management/commands/update_project_statuses.py

```diff
@@ -32,19 +32,36 @@ def add_arguments(self, parser):
             type=valid_date,
             help="Start date for project status updates (default: 6 months before current date, format: YYYY-MM-DD)."
         )
+        parser.add_argument(
+            '--date-end',
+            dest="date_end",
+            default=datetime.now(),
+            type=valid_date,
+            help="End date for project status updates (default: today, format: YYYY-MM-DD)."
+        )
 
     def handle(self, *args, **options):
         self.options = options
 
+        build_count = Build.objects.filter(
```

Comment:
isn't this redundant? It's the same as variable 'total' below.

Reply:
Thanks, fixed.